Week1/homework/homework.txt

```diff
@@ -0,0 +1,81 @@
+Here is the lucidChart diagram:
+https://www.lucidchart.com/documents/edit/00de9f5d-4578-420f-8fed-93e057398424/0
```

Comment:
I tried to use the link, but I don't have access to it. Could you take a screenshot of it instead and push the image to git? Thanks!

Reply:
Let's see if this works - if not, I will use an image.

https://www.lucidchart.com/invitations/accept/9ed51ccc-14fb-470c-ac92-2d5cd9ec2440